docs/releases/1.9.txt

```diff
@@ -151,6 +151,11 @@ Minor features
 * The time picker widget includes a '6 p.m' option for consistency of having
   predefined options every 6 hours.
 
+:mod:`django.contrib.admin`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Admin Docs now also display model's methods that take arguments.
```

Comment:
I'd mimic what you wrote before, "The models section now includes methods that take arguments."